cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -61,6 +66,10 @@ TEST(RayClusterModeTest, FullTest) {
   auto get_result = *(Ray::Get(obj));
   EXPECT_EQ(12345, get_result);
 
+  auto named_obj =
+      Ray::Task(Return1).SetName("named_task").SetResources({{"CPU", 1.0}}).Remote();
```

Comment:
How do we know the name and resources work?

Reply:
Have added a test case for resources.